allennlp/common/plugins.py

```diff
@@ -66,9 +67,10 @@ def discover_plugins() -> Iterable[str]:
     """
     Returns an iterable of the plugins found.
     """
-    for module_info in discover_namespace_plugins():
-        yield module_info.name
-    yield from discover_file_plugins()
+    with push_python_path("."):
```

Comment:
What does this do when I run something from some unexpected working directory?

Reply:
What do you mean? This pushes a path string into `sys.path`.

Reply:
Yes, I get it now.